.travis.yml

```diff
@@ -2,6 +2,8 @@
 # RVM/bundler/ruby and whatnot. Right now 'rust' as a language actually
 # downloads a rust/cargo snapshot, which we don't really want for building rust.
 language: c
+compiler: /usr/bin/gcc-4.7
```

Comment:
Right but the intention is not clear. This is setting a pretty arbitrary key in this configuration file, "compiler", which has no connection to the CC environment variable and the only reason we want to do this is to set the CC environment variable. I'd prefer to explicitly do so in the `env` array.